marklogic-data-hub/src/test/java/com/marklogic/hub/flow/FlowRunnerTest.java

```diff
@@ -99,6 +101,11 @@ public void testRunFlow(){
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_FINAL_NAME, "json-map") == 1);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_FINAL_NAME, "xml-map") == 1);
         Assertions.assertTrue(JobStatus.FINISHED.toString().equalsIgnoreCase(resp.getJobStatus()));
+        XMLDocumentManager docMgr = stagingClient.newXMLDocumentManager();
```

Comment:
The change looks good, but can you change this into a unit test that hits applyPermissions directly? You'll need to make that method protected, which I think is fine for the sake of unit testing.

I say this because I'm about to dig into why this test method is failing in Jenkins but nowhere else, so I don't want to see anything added to this test as something is fishy with it already. A unit test is much faster to run as well.

Reply:
@rjrudin ,
I can change it to a unit test but this PR is for 5.1-develop where all tests are passing. I haven't made a PR for develop.